contracts/modules/royalty/policies/RoyaltyPolicyLAP.sol

```diff
@@ -230,6 +240,7 @@ contract RoyaltyPolicyLAP is
             uniqueParents[index] = parentIpIds[i];
             _setRoyalty(ipId, parentIpIds[i], royaltiesGroupByParent[index]);
         }
+        IP_GRAPH_ACL.disallow();
```

Comment:
the disallow needs to be moved lower because `_getRoyaltyStack` and `_getAncestorCount` are also precompiles no?

Reply:
nope, as only the write functions (addParent, setRoyalty) of IPGraph require access control.